src/Plugin.php

```diff
@@ -46,6 +46,13 @@ public static function init() {
     // Allow SVG files in media library.
     add_filter('upload_mimes', __CLASS__ . '::upload_mime_types');
 
+    // Remove size attributes from SVG images, so they scale correctly without a CSS size reset.
```

Comment:
@sun - I changed the comment here and adjusted the commit description. Thanks for reviewing!